src/volt/components/ProposalsList/index.js

```diff
@@ -24,4 +23,4 @@ const ProposalsList = props => {
   );
 };
 
-export default ProposalsList;
+export default ProposalsList
```

Comment:
??? :-)

Reply:
Prettifier?
Again, it was before your PR...